android/build.gradle

```diff
@@ -37,8 +37,8 @@ project.afterEvaluate {
 }
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion '23.0.2'
+    compileSdkVersion 24
```

Comment:
I updated `.travis.yml`. Let me know if there's any problem with that. Hopefully Travis will built it properly now.

(I left `com.android.tools.build:gradle:2.1.3` since the command-line gradlew couldn't load build tools 2.2.0 in my BBQ Timer app. Maven central doesn't have it yet?)

I'll remove the large icons from the notifications on N.